subscriber_queue_length: 100
subscribe_to_odom_topic: "/firefly/ground_truth/odometry"
publish_to_tf_frame_id: "simulated_odometry"
x_vel_mean: 0
x_vel_stddev: 0
y_vel_mean: 0
y_vel_stddev: 0
z_vel_mean: 0
z_vel_stddev: 0
yaw_rate_mean: 0
yaw_rate_stddev: 0
<!--x_mean: 0-->
<!--x_stddev: 0-->
<!--y_mean: 0-->
<!--y_stddev: 0-->
<!--z_mean: 0-->
<!--z_stddev: 0-->
<!--yaw_mean: 0-->
<!--yaw_stddev: 0-->
<!--pitch_mean: 0-->
<!--pitch_stddev: 0-->
<!--roll_mean: 0-->
<!--roll_stddev: 0-->